### PR TITLE
Adds pull request template. See template below. 

### PR DESCRIPTION
**Pull request by jackblanc**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)

## Verification Steps

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Why
+
+[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)
+
+## This PR
+
+## Screenshots
+
+## Verification Steps
```
Comment (rymaju) [inline .github/pull_request_template.md]: Verification steps?

Comment (rymaju) [reply .github/pull_request_template.md]: Can we also add some text as instructions on what exactly should go in each section?

Comment (jackblanc) [reply .github/pull_request_template.md]: Yeah, wasn't sure if it's best to include here or explain elsewhere

**Review by rymaju — APPROVED** [review]
Very Nice 👌

**Pull request merged**